build/memory_tests.ps1

```diff
@@ -0,0 +1,6 @@
+$nunitrunner = "C:/Tools/NUnit3/nunit3-console.exe";
```

Comment:
It would be nice to use an env variable for nunit 3 console path, to be able to reuse this script (ie: locally) regardless of the path.